packages/@react-spectrum/s2/intl/ar-AE.json

```diff
@@ -1,4 +1,5 @@
 {
+  "breadcrumbs.more": "More items",
   "button.pending": "قيد الانتظار",
```

Comment:
Some locales are in English, which is ok, just have not recevied translation yet, will be updated once we get translation in